### DISCUSSION TITLE
Help connecting to Ollama instance (unRAID > MBP)

### DISCUSSION THREAD

**Discussion by Sptzzz** [open-webui/open-webui#934] [Q&A]
Hi all!

So I've tried both ollama and webgui in unRAID and they work perfectly, but my GPU only has 6GB Ram so it's quite limited and using CPU (5800x) is slow as hell.

Since I have an M3 MBP I tried locally and it runs miles better so I wanted to host the webgui on unRAID and connect to my ollama instance in my MBP.

As you can see in the screenshot I've put in the correct IP and of course ollama is running and localhost:11434 shows its running. But I'm always getting a "cant connect, make sure it's updated etc". It worked perfectly when I also hosted ollama on unraid. So I'm a bit at a loss on how to go from here?

![CleanShot 2024-02-27 at 09 57 15@2x](https://github.com/open-webui/open-webui/assets/159544564/e8ce4a15-67a7-4415-9d9a-12a08b165117)

Reasoning for this is because I want people in my network to access the webgui!

Thank you!

**Comment by justinh-rahb** [comment]
Read Ollama FAQ: [setting environment variables](https://github.com/ollama/ollama/blob/main/docs/faq.md#setting-environment-variables-on-mac) (specifically `OLLAMA_HOST=0.0.0.0`)

Then ours: [Installing with Docker and Ollama on a different server](https://docs.openwebui.com/getting-started/#using-ollama-on-a-different-server) (setting `OLLAMA_API_BASE_URL=http://<your_macs_ip_address>:11434/api` environment variable for container)
